apis/apiextensions/v1alpha1/composition_types.go

```diff
@@ -48,14 +48,14 @@ var (
 
 // CompositionSpec specifies the desired state of the definition.
 type CompositionSpec struct {
-	// From refers to the type that this composition is compatible. The values
-	// for the underlying resources will be fetched from the instances of the
-	// From.
+	// CompositeTypeRef refers to the type that this composition is compatible.
+	// The values for the underlying resources will be fetched from the
+	// instances of the referenced composite type.
```

Comment:
```suggestion
	// CompositeTypeRef specifies the type of composite resource that this
	// composition is compatible with.
```

I'd prefer to remove the "values will be fetched from the composite" part while we're in here, since we're planning to allow bidirectional patching per https://github.com/crossplane/crossplane/issues/1639.